src/datastores/sql/crud/folder.py

```diff
@@ -335,23 +336,66 @@ def update_folder_in_db(db: Session, folder: schemas.FolderUpdateRequest):
 
 
 def delete_folder_from_db(db: Session, folder_id: int):
```

Comment:
I assume it's intended that this method does not return anything? nit: add return type None to signature

Reply:
Done